docs/troubleshooting.md

```diff
@@ -12,21 +12,33 @@ bin/doctor
 
 ### Diagnose general issues with a project/app not working
 
+* Make sure you run all commands via GOV.UK Docker.
+
 ```
-# make sure GOV.UK Docker is up-to-date
-git pull
+govuk-docker-run bundle install
+govuk-docker-run rake db:migrate
+```
+
+* Check if one of the dependencies is the problem.
 
-# make sure the project is built OK
-make <project>
+> A common problem for dependencies is when you've previously `git pull`ed the repo, but haven't run `bundle install` or `rake db:migrate`. The logs for the dependency will show if this is the problem.
 
+```
 # check if any dependencies have exited
 docker ps -a
 
-# tail logs for running services/dependencies
+# check logs for an exited dependency
 govuk-docker logs -f publishing-api-app
+```
+
+* Try cleaning up and running your command again.
+
+```
+# stop all apps and their dependencies
+govuk-docker down
 
-# try clearing all containers / volumes
-govuk-docker rm -sv
```

Comment:
I know you want to encourage people to diagnose and fix errors 'properly', but sometimes if you're in a bind and need to get something working quickly, this is just the easiest option. I'd really like you to leave it in, even if it's caveated with "You really shouldn't have to do this...!"

Reply:
I agree there is a need to have a clean slate. The `govuk-docker down` command I've suggested is actually equivalent to this, but has the advantage of being easier to explain and write down.

Does that help address your concern?

Reply:
Sure, thanks for explaining.